psydac/linalg/basic.py

```diff
@@ -196,6 +196,11 @@ def dtype(self):
     def T(self):
         return self.transpose()
 
+    @property
+    def H(self):
+        L = self.T
+        return L.conjugate(out=L)
```

Comment:
Sorry if I was not clear in my previous explanation. We cannot provide an implementation for this property based on the method `conjugate`, because such a method is not defined by the base class (in other words, there is no method called `conjugate` here). I will fix it myself.

Reply:
This should be fixed by b39734a.